.nvmrc

```diff
@@ -1 +1 @@
-16
\ No newline at end of file
+18
```

Comment:
18 is the latest LTS version. Didn't experience any conflicts with dependencies.